tests/acceptance/button-test.js

```diff
@@ -3,17 +3,21 @@ import { module, test } from 'qunit';
 import startApp from '../helpers/start-app';
 import contains from '../helpers/contains';
 
-var App, AppController;
+var App, AppController, DisabledController;
```

Comment:
this feels like it should be in its own test file `tests/acceptance/disabled-button-test.js` maybe?